src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -81,6 +87,14 @@ public class TypicalPersons {
             .withEmail(VALID_EMAIL_BOB).withAddress(VALID_ADDRESS_BOB)
             .withTags(VALID_TAG_HUSBAND, VALID_TAG_FRIEND)
             .build();
+    public static final Person Erza = new PersonBuilder().withName(VALID_NAME_ERZA).withPhone(VALID_PHONE_ERZA)
```

Comment:
```suggestion
    public static final Person ERZA = new PersonBuilder().withName(VALID_NAME_ERZA).withPhone(VALID_PHONE_ERZA)
```

might want to capitalize